pancancer_evaluation/utilities/classify_utilities.py

```diff
@@ -770,26 +832,37 @@ def shuffle_by_cancer_type(y_df, seed):
 def apply_model_params(train_model,
                        ridge=False,
                        lasso=False,
-                       lasso_penalty=None):
+                       lasso_penalty=None,
+                       model='lr'):
```

Comment:
Do you make a comment of which values are accepted for `model`? Looks like it's just lr vs. mlp, but it could be good to have a comment explaining that lr encompasses ridge and lasso.

Reply:
Good idea! I'll add a comment.